plutus-tx-template/README.md

```
@@ -0,0 +1,156 @@
+# 📦 Auction Validator Project
+
+### Updated by Coxygen Global - Bernard Sibanda
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)